### PR TITLE
update Chinese translation zh_CN

### PR DESCRIPTION
**Pull request by syxunion**

### CODE REVIEW

File: translations/zh-CN.json [hunk 1]
```diff
@@ -1082,30 +1082,30 @@
...
-	"You cannot change the number of nodes for an existing minikube cluster. Please use 'minikube node add' to add nodes to an existing cluster.": "",
+	"You cannot change the number of nodes for an existing minikube cluster. Please use 'minikube node add' to add nodes to an existing cluster.": "不能更改现有 minikube 集群的节点数。请使用 'minikube node add' 向现有集群添加节点。",
 	"You cannot change the static IP of an existing minikube cluster. Please first delete the cluster.": "您不能更改现有 minikube 集群的静态 IP。请先删除集群。",
-	"You cannot enable addons on a cluster without Kubernetes, to enable Kubernetes on your cluster, run: minikube start --kubernetes-version=stable": "",
-	"You have authenticated with a service account that does not have an associated JSON file. The GCP Auth addon requires credentials with a JSON file in order to continue.": "",
-	"You have chosen to disable the CNI but the \"{{.name}}\" container runtime requires CNI": "",
+	"You cannot enable addons on a cluster without Kubernetes, to enable Kubernetes on your cluster, run: minikube start --kubernetes-version=stable": "你不能在没有 Kubernetes 的集群上启用插件，要在你的集群上启用 Kubernetes，运行:minikube start --kubernetes-version=stable",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:1091]:
```suggestion
	"You cannot enable addons on a cluster without Kubernetes, to enable Kubernetes on your cluster, run: minikube start --kubernetes-version=stable": "您不能在没有 Kubernetes 的集群上启用插件，要在你的集群上启用 Kubernetes，运行:minikube start --kubernetes-version=stable",
```

File: translations/zh-CN.json [hunk 2]
```diff
@@ -1082,30 +1082,30 @@
 	"You can create one using 'minikube start'.\n\t\t": "您可以使用 'minikube start' 创建一个.\n\t\t",
 	"You can delete them using the following command(s): ": "您可以使用以下命令删除他们：",
 	"You can force an unsupported Kubernetes version via the --force flag": "你可以通过 --force 标志强制使用不支持的 Kubernetes 版本",
-	"You cannot add or remove extra disks for an existing minikube cluster. Please first delete the cluster.": "",
-	"You cannot change the CPUs for an existing minikube cluster. Please first delete the cluster.": "",
-	"You cannot change the disk size for an existing minikube cluster. Please first delete the cluster.": "",
+	"You cannot add or remove extra disks for an existing minikube cluster. Please first delete the cluster.": "不能为已存在的 minikube 集群添加或删除额外的磁盘。请先删除集群。",
+	"You cannot change the CPUs for an existing minikube cluster. Please first delete the cluster.": "不能对已存在的 minikube 集群修改 CPU。请先删除集群。",
+	"You cannot change the disk size for an existing minikube cluster. Please first delete the cluster.": "不能更改现有 minikube 集群的磁盘大小。请先删除集群。",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:1087]:
```suggestion
	"You cannot change the disk size for an existing minikube cluster. Please first delete the cluster.": "您不能更改现有 minikube 集群的磁盘大小。请先删除集群。",
```

上面两个也一样,缺少了 "您"

Comment (syxunion) [reply translations/zh-CN.json]: done

**Review by medyagh — APPROVED** [review]

**Pull request merged**